### PR TITLE
Add support for floating numbers

### PR DESCRIPTION
**Pull request by jni**
Hey cool tool, I just need this one change for it to work for my use case. Can you accept it?

### CODE REVIEW

**Comment by STATSmonash** [comment]
yes that is great.  but i only need int.  why don't you create a new function and i will copy that.

**Comment by jni** [comment]
It feels a bit redundant to have two functions. Floats should still work to add ints together. What do you think?

File: calc.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # '1 + 2' -> 3, or '1 - 2' -> -1.
 def compute(expression):
     num0, operator, num1 = expression.split(' ')
-    num0, num1 = int(num0), int(num1)
+    num0, num1 = float(num0), int(num1)
```
Comment (STATSmonash) [inline calc.py]: why is num1 still an int?  change this to float and i will accept your changes

Comment (jni) [reply calc.py]: @STATSmonash done! Thanks for the review!

**Pull request merged**